test/integration/acl-tls-test.js

```diff
@@ -88,11 +88,15 @@ describe('ACL with WebID+TLS', function () {
   }
 
   describe('no ACL', function () {
-    it('should return 500 for any resource', function (done) {
+    before(() => {
+      rm(path.join(rootPath, '.acl'))
+    })
+
+    it('should return 403 for any resource', function (done) {
       var options = createOptions('/acl-tls/no-acl/', 'user1')
       request(options, function (error, response, body) {
         assert.equal(error, null)
-        assert.equal(response.statusCode, 500)
```

Comment:
Actually, I think we shouldn't change this test, because it is a server misconfiguration to not have a root ACL. So, here it is mostly just that it became a tad harder to test. Since it is covered by a test in acl-oidc-test, if we can't easily fix the test, I think we should just skip it for now.